supermarq/converters.py

```diff
@@ -4,9 +4,10 @@
 import qiskit
 
 
-def cirq_to_qiskit(  # pylint: disable=missing-function-docstring
+def cirq_to_qiskit(
     circuit: cirq.Circuit,
 ) -> qiskit.circuit.QuantumCircuit:
+    """Convert a circuit from `cirq` to `qiskit`."""
```

Comment:
Why is pylint letting you get away with specifying `Args` and `Returns` here? CC @pgoiporia

Reply:
the current checks don't occur if there's no "Args:"/"Returns:"/"Raises:"/etc in the docstring. To require these sections be present whenever applicable, we also have to add these to setup.cfg:
```
accept-no-param-doc=no
accept-no-raise-doc=no
accept-no-return-doc=no
accept-no-yields-doc=no
```

Reply:
Do we want to always explicitly specify the args / returns / raises when it's extremely obvious what they represent? I've basically been sort of leaving them blank unless it's not super obvious (for ex. a variable name/return type is not descriptive)

Reply:
was wondering this as well. on one hand the consistency is nice, and combined with the checker it forces us to update the docstring if e.g. we add a new argument to a function. but it's also hard to say something like
```python
def qiskit_gate_to_cirq_gate(qiskit_gate: qiskit.circuit.Gate) -> cirq.Gate:
```
really needs much explanation

i think overall i'm in favor of requiring them always: i don't think there are \*that\* many cases where they aren't at least a little bit helpful, and it'd be nice to be able to have pylint require them so that they're never missing in places they actually are useful. could definitely be convinced otherwise though

Reply:
yeah I agree, and it might even be useful in the simpler cases. I can see a situation where someone might change the return type, without also having to update the docstrings to represent the change it's easy to let hacky solutions slip through the cracks